parm/config/config.wave

```diff
@@ -20,18 +20,22 @@ export COMPONENTwave=${COMPONENTwave:-${RUN}wave}
 export COMPONENTRSTwave=${COMPONENTRSTwave:-gdaswave}
 
 # Grids for wave model
-export waveGRD='glo_10m aoc_9km ant_9km'
+# GFSv16
+export waveGRD='gnh_10m aoc_9km gsh_15m'
 export waveGRDN='1 2 3' # gridnumber for ww3_multi
 export waveGRDG='10 20 30' # gridgroup for ww3_multi
 
 # ESMF input grid
 export waveesmfGRD='glox_10m' # input grid
+#export waveesmfGRD='glo_15mxt' # input grid
```

Comment:
My preference is to not retain commented out variable settings.   If the setting is commented out  and not used, remove the unused setting from the config file.

Reply:
I agree. This has been changed and will appear in a forthcoming push.

Reply:
Done